README.md

```diff
@@ -5,11 +5,18 @@ This is a project template providing a great out-of-the-box experience for new D
 
 This project includes the following packages and configuration:
 * Drupal core
-* [Acquia CMS](https://github.com/acquia/acquia_cms) (Drupal distribution)
+* Drupal core scaffold
```

Comment:
Let's link this to the project like the others

Reply:
Done.